store/src/main/java/com/dropbox/android/external/store4/impl/RealStore.kt

```diff
@@ -60,10 +62,10 @@ internal class RealStore<Key : Any, Input : Any, Output : Any>(
     private val memCache = memoryPolicy?.let {
```

Comment:
As mentioned above add @OptIn(ExperimentalTime::class) here and remove the class level `@ExperimentalTime` so the rest of `store` module don't need to propagate this annotation.

Reply:
Because I put the annotation on the Cache interface, functions calls on memCache also require ExperimentalTime. So wouldn't it be better to have ExperimentalTime on expireAfterAccess and expireAfterWrite as before?

Reply:
Can leave it as is. See comment above.

Reply:
If I leave the annotation on `Cache`, then `RealStore` need `OptIn` on `stream()`, `clear()` and `clearAll()`.

Reply:
And because `RealStore` takes in `MemoryPolicy` in the constructor, then it also needs `OptIn`. I can put `ExperimentalTime` on the constructor of `MemoryPolicy` instead of the class and on the fields. Then we can avoid RealStore from needing any `OptIn` besides on `memCache`.

Reply:
I have pushed the change so you can easier see what I mean.

Reply:
Can't we just add a `@ExperimentalTime` to `internal class RealStore`?

Reply:
We can also avoid `ExperimentalTime` in the filesystem module, if we put `ExperimentalTime` on the function calls in `CacheBuilderImpl` instead of the class.

Reply:
👍

Reply:
> Can't we just add a `@ExperimentalTime` to `internal class RealStore`?

You are right. Done.

Reply:
> We can also avoid `ExperimentalTime` in the filesystem module, if we put `ExperimentalTime` on the function calls in `CacheBuilderImpl` instead of the class.

But we can't do it if `Cache` has `ExperimentalTime`.